pysatSpaceWeather/instruments/methods/ace.py

```diff
@@ -331,3 +346,75 @@ def load_csv_data(fnames, read_csv_kwargs=None):
 
     data = pds.DataFrame() if len(fdata) == 0 else pds.concat(fdata, axis=0)
     return data
+
+
+def ace_swepam_hourly_omni_norm(as_inst, speed_key='sw_bulk_speed',
+                                dens_key='sw_proton_dens',
+                                temp_key='sw_ion_temp'):
+    """Normalize ACE SWEPAM variables as described in the OMNI processing _[1].
+
+    Parameters
+    ----------
+    as_inst : pysat.Instrument
+        pysat Instrument object with ACE SWEPAM data
```

Comment:
```suggestion
        pysat Instrument object with ACE SWEPAM data.
```

Reply:
Not sure I completely grasp when periods are appropriate, but it looks like this function's parameter descriptions could use periods.

Reply:
Not sure I completely grasp when periods are appropriate, but it looks like this function's parameter descriptions could use periods.

Reply:
It's not necessary here, but I'll add the suggested changes.